mission/src/store/index.js

```diff
@@ -1,8 +1,49 @@
 import { createStore } from 'vuex';
+import Repository from '@/repositories/RepositoryFactory';
+
+const CartRepository = Repository.get('cart');
 
 export default createStore({
-  state: {},
-  mutations: {},
-  actions: {},
+  state() {
+    return {
+      siteName: 'DefJam Streetwear 🥾',
+      cartList: [],
+    };
+  },
+
+  getters: {
+    cartTotalPrice(state) {
+      let sum = 0;
+      for (let i = 0; i < state.cartList.length; i += 1) {
+        sum += state.cartList[i].price;
+      }
+      return sum;
+    },
```

Comment:
반복문은 forEach 메서드 사용을 고려해보시는 것도 좋을 것 같아요 :)
선호도 차이라고 하는데 개인적으로 중간에 break가 필요한 조건이 없다면 좀더 간결해 보이더라구요!

Reply:
다애님, 오랜만이네요! 지난 5주 동안 다애님도 정말 수고 많으셨어요 :-D

여기 제안해주신 내용은 바로 적용해볼 수 있을 것 같아요!! 아래와 같이 `forEach`로 구현하는 걸 말씀하신 거죠?
아무래도 제가 쉽고 익숙한 메서드를 써 버릇하는 경향이 있어서 별 고민 없이 `for`을 사용했던 것 같네요 😆

코드 라인도 한 줄로 줄어들어 확실히 코드가 훨씬 간결해지고 좋은 것 같습니다 👍👍

```suggestion
    cartTotalPrice(state) {
      let sum = 0;
      state.cartList.forEach((item) => { sum += item.price; });
      return sum;
    },
```